src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -231,6 +232,7 @@ public function configureOptions(OptionsResolver $resolver)
             // is manually set to an object.
             // See https://github.com/symfony/symfony/pull/5582
             'data_class' => null,
+            'trans_options' => true,
```

Comment:
i think it should be named `translate_choices`
1. we don't abbreviate option names AFAIK, e.g. preferred
2. `choices` is also the term in the related option